internal/handlers/common/aggregations/operators/operators.go

```diff
@@ -16,24 +16,53 @@
 package operators
 
 import (
-	"context"
+	"fmt"
 
 	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
 )
 
-// newAccumulatorFunc is a type for a function that creates an accumulation operator.
-type newAccumulatorFunc func(expression *types.Document) (Accumulator, error)
+// newOperatorFunc is a type for a function that creates a standard aggregation operator.
```

Comment:
Could you describe what is standard aggregation operator? I think example would be helpful here.